src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+
+const titles = [...document.querySelectorAll('ul')];
```

Comment:
Consider using more descriptive variable names instead of `titles`. For example, `listItems` or `unorderedLists`. 📝